### ISSUE TITLE
Some glbs load slowly

### ISSUE THREAD

**Issue by ryantrem** [BabylonJS/BabylonReactNative#72]
I haven't investigated this at all, only observed the issue and didn't want to lose track of it. For this model:
https://raw.githubusercontent.com/microsoft/experimental-pcf-control-assets/master/surface.glb

Loading it in SceneForm directly is pretty much instantaneous.
Loading it in Babylon React Native with JSC takes around 2 seconds.
Loading it in Babylon React Native with Hermes takes around 5 seconds.

The above times are similar regardless of debug or release.

**Comment by bghgary** [comment]
This loads very fast on the web. I don't see anything in this model that will make it slow to load. The only thing I can think of that can be slow is the mipmap generation which should only be slow in debug.

**Comment by ryantrem** [comment]
On iOS/JSC (on my iPhone X), this model actually takes 9 seconds to load.

**Comment by ryantrem** [comment]
Revising my numbers based on an actual release build instead of a debug build 🤦🏼‍♂️  (the numbers below are all on an iPhone X):
- Loading it in SceneKit: ~800ms on the first load, ~350ms until the app is restarted
- Loading it in Babylon React Native with JSC: ~1.9s first load and subsequent loads
- Loading it in Babylon.js (in Safari): ~600ms first load, ~300ms until the browser cache is cleared

So in summary, load time is about double in Babylon Native compared to Babylon.js or directly in SceneKit. Not great, but not an order of magnitude, so I'd say this issue is lower priority than initially thought.

**Comment by ryantrem** [comment]
React Native's `XMLHttpRequest` takes the byte buffer, converts it to a base64 string, passes that string (by copy) to JavaScript, and finally it is converted in JavaScript back to a byte buffer. This is very slow. This has been addressed by allowing Babylon.js to use Babylon Native's `XMLHttpRequest` implementation instead of React Native's. See these PRs for details:
- https://github.com/BabylonJS/Babylon.js/pull/9148
- https://github.com/BabylonJS/BabylonNative/pull/442
- https://github.com/BabylonJS/BabylonReactNative/pull/94
